.github/workflows/require-allow-edits.yml

```diff
@@ -0,0 +1,14 @@
+name: Require “Allow Edits”
+
+on: [pull_request_target]
```

Comment:
Let's use `pull_request` if write permission is not required. This way we'll also see in this PR if there are no syntax errors.